### ISSUE TITLE
Show interconnections between resources on list mode

### ISSUE THREAD

**Issue by Garbash** [CyberCRI/KnowNodes#78]

**Comment by deemeetree** [comment]
As in, if I'm on a resource A and it's connected to B, C and D and C and D are also connected then that connection is shown? 

Same for listing the recent connections and for user's connections?

**Comment by Garbash** [comment]
Exactly, and it would be great if it would apply to resource centered pages
as well front page, profile page etc', but I think implementation on the
latter would be probably difficult

On Fri, Oct 11, 2013 at 9:10 PM, Dmitry Paranyushkin <
notifications@github.com> wrote:

> As in, if I'm on a resource A and it's connected to B, C and D and C and D
> are also connected then that connection is shown?
> 
> Same for listing the recent connections and for user's connections?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/CyberCRI/KnowNodes/issues/78#issuecomment-26163559
> .

## 

Get some insight of what makes me tick:
My twitter https://twitter.com/#!/garbash
Knownodes mailing list https://groups.google.com/forum/#!forum/knownodes

Dor Garbash
108 Rue du faubourg du temple, Paris 75011
+33-652919878

**Comment by deemeetree** [comment]
Ok, the easiest would be to just add another pane at the bottom of the page, under the area where all the connections to the main resource are shown. That new pane could be called Shared Connections and it could list in the same design all the connections among the resources connected to the main one.

Is that what you were thinking about?

Another solution could be to present information not in terms of successive connections (in this case we're operating with pairs) but in terms of triads – if A is the main resource connected to B C and D (where C and D as well as B and D are also connected), then we identify first the nodes that have the most mutual neighbors to the main resource (D shares B and C with A, B shares only D, and C shares only D). So in this case the node with the highest number of mutual neighbors (D=2) is placed in the middle of the 3 and there is the next one in the number of mutual neighbors on top of it (e.g. C=1 as the most recent one) and B=1 under. If there was one more node E that was only connected to A and nothing else, then it would go last in that list.

I could think of a Cypher query suitable for that, which would also sort the nodes accordingly within the query itself (unless we want to make it separately).
